Scuti/Plugins/iOS/NativeAppstore.mm

```diff
@@ -0,0 +1,57 @@
+#import "NativeAppstore.h"
```

Comment:
@cdga777 Can we either include this plugin as a dependency to our plugin using the dependencies property in package.json?  Or rename so it doesn't collide with someone else using this plugin?

Reply:
@markgrossnickle I'm not sure how to do these :sweat:

Reply:
@cdga777 Maybe just add a namespace for now to avoid conflicts